.github/workflows/code-review-gpt.yml

```diff
@@ -0,0 +1,33 @@
+name: Code Review GPT
+
+on:
+  pull_request:
+    branches: [main]
+
+permissions:
+  pull-requests: write
```

Comment:
Any chance to narrow down to e.g. writing just **comments** on PRs?

Reply:
I guess so. we could add code-review-gpt to our docker image to boos the execution.